docs/guides/admin/docs/workflowoperationhandlers/statistics-writer.md

```diff
@@ -0,0 +1,48 @@
+Statistics Writer Workflow Operation
+====================================
+
+ID: `statistics-writer`
+
+Description
+-----------
+
+The statistics writer operation can be used to publish statistics about a video to a statistics backend such as
+InfluxDB. Currently, it only writes the length of the video in seconds to the data base. It can be configured to
+write the negative length and can thus be used for retract workflows, too.
+
+
+Parameter Table
+---------------
+
+|configuration keys            |required|description                                                        |
+|------------------------------|--------|-------------------------------------------------------------------|
+|flavor                        |yes     |The flavor of the track you want to publish statistics to          |
```

Comment:
The flavor of the track you want to publish statistics about, I suspect.

Reply:
Yep, sounds better, fixed.